examples/petsc/qfunctions/bps/bp3.okl

```diff
@@ -89,25 +91,32 @@ typedef double CeedScalar;
 
     const CeedScalar c[3] = { 0, 1., 2. };
     const CeedScalar k[3] = { 1., 2., 3. };
+
     out[oOf7[0]+i] = sin(M_PI*(c[0] + k[0]*in[iOf7[0]+0*Q+i])) *
-                       sin(M_PI*(c[1] + k[1]*in[iOf7[0]+1*Q+i])) *
-                       sin(M_PI*(c[2] + k[2]*in[iOf7[0]+2*Q+i]));
+                     sin(M_PI*(c[1] + k[1]*in[iOf7[0]+1*Q+i])) *
+                     sin(M_PI*(c[2] + k[2]*in[iOf7[0]+2*Q+i]));
```

Comment:
It's not the issue, but `M_PI` doesn't have a `#def` in `bp3.okl` or `bp4.okl`. I'm not sure where OCCA getting the definition of `M_PI`.

Reply:
Yeah, I was also wondering where it gets it. But yes, it is not the issue here. Thanks